Adding selenium script to ServiceNow (basic auth)

tyler-cloud-elements opened the pull request:
## Highlights
* Added selenium script to ServiceNow

knappkeith commented on src/test/elements/servicenow/assets/scripts.js:
```diff
@@ -0,0 +1,34 @@
+      browser.get("https://developer.servicenow.com/app.do#!/instance");
+      browser.wait(webdriver.until.elementLocated(webdriver.By.id('username')), 10000);
+      browser.findElement(webdriver.By.id('username')).clear();
+      browser.findElement(webdriver.By.id('username')).sendKeys('developer@cloud-elements.com');
```
@tyler-cloud-elements, Can you move this and the password to churros-sauce.

knappkeith commented on src/test/elements/servicenow/assets/scripts.js:
```diff
@@ -0,0 +1,61 @@
+      browser.findElement(webdriver.By.id('submitButton')).click();
+      browser.sleep(5000);
+      browser.get(wakeUpInstanceUrl);
+      browser.sleep(5000);
```
Are the `sleeps` absolutely needed?  In this method I counted 16.5 seconds in explicit sleeps, can `.wait` methods be used instead?

knappkeith commented on src/test/elements/servicenowoauth/assets/scripts.js:
```diff
@@ -5,30 +5,39 @@ const props = require('core/props');
-      browser.get("https://developer.servicenow.com/app.do#!/instance");
-      browser.sleep(5000);
+      browser.get(wakeUpInstanceUrl);
+      browser.sleep(10000);
```
Same as for `servicenow/assets`

tyler-cloud-elements commented:
@knappkeith I removed any unnecessary sleeps where possible, but the ones remaining are needed, (i.e. giving time for the authentication, but not loading the page). We are still using the `.waits` where we can, but ServiceNow is unreliable so we've added the browser sleeps as buffer. For example, the page to show whether the instance asleep can take anywhere from 2 - 15 seconds to show up. So now we should catch it in all occurrences. 

General time info:
- If it needs to wake up the instance, it can take up to 3 minutes before running the test
- If not, it will take apx 15+ seconds (depending on ServiceNow) before running the test
